### ISSUE TITLE
Failing to run KH 1.5 + 2.5 with version 2.3.0

### ISSUE THREAD

**Issue by arcogabbo** [HansKristian-Work/vkd3d-proton#643]
Installed the latest release and whenever i run the game i get this output:
```
001b:err:ntoskrnl:ZwLoadDriver failed to create driver L"\\Registry\\Machine\\System\\CurrentControlSet\\Services\\NDIS": 00000001
0035:err:ole:CoGetContextToken apartment not initialised
51:info:vkd3d_get_vk_version: vkd3d-proton - applicationVersion: 2.3.0.
51:info:vkd3d_instance_init: vkd3d-proton - build: 26584b4d7cee8990.
51:err:vkd3d_load_vk_instance_procs: Could not get instance proc addr for 'vkGetPhysicalDeviceFragmentShadingRatesKHR'.
51:err:vkd3d_instance_init: Failed to load instance procs, hr 0x80004005.
0031:err:ntdll:RtlpWaitForCriticalSection section 0x6427eeb2778 "?" wait timed out in thread 0031, blocked by 0000, retrying (60 sec)
```

**Comment by DadSchoorse** [comment]
Can you provide the output of `vulkaninfo`? (most distros have a vulkan-tools package for that)

**Comment by arcogabbo** [comment]
Here it is https://ghostbin.co/paste/6yqrm

**Comment by DadSchoorse** [comment]
And what wine version are you using?

**Comment by arcogabbo** [comment]
Well, 5.0. I guess the problem is the version, in fact lutris is using 6.4 and the game boots but i get a page fault on read access.

**Comment by DadSchoorse** [comment]
I think #644 should fix the original issue, no idea about the segfault. A vkd3d-proton log might help.

**Comment by DadSchoorse** [comment]
actually, in the page fault info there's
> Version: Windows Server 2003
you should set the windows version in the prefix to windows 10

**Comment by arcogabbo** [comment]
Sasdly, the same page fault occurs setting Windows version to 10

**Comment by robzombie91** [comment]
this is the bug i get repeatedly

340:fixme:d3d12_command_list_update_current_framebuffer: Invalid RTV for attachment 0.
340:fixme:d3d12_command_list_ExecuteBundle: iface 00000000057492d0, command_list 0000000005aaa2d0 stub!
344:fixme:d3d12_swapchain_Present1: Ignored present parameters 000000000ed7fba8.

**Comment by DadSchoorse** [comment]
@robzombie91 you are not using 2.3m d3d12_command_list_ExecuteBundle was implemented recently.

**Comment by Joshua-Ashton** [comment]
This should be fixed in latest master.

**Comment by Joshua-Ashton** [comment]
Also, you probably want to update your wine at some point. 5.0 is pretty old.

**Comment by arcogabbo** [comment]
Yeah i installed an updated version with lutris but i always encounter a page fault, i saw somewhere wine 6.7 Is releasing soon, i'll try it and see.

**Comment by DistantThunder** [comment]
I went a bit further trying to run this game and fell upon a most strange happening. 

First on 2.3.0, I'm still getting lots of : 
> MANGOHUD: recreating font image
328:fixme:d3d12_swapchain_Present1: Ignored present parameters 000000000fbafb58.
332:fixme:d3d12_swapchain_Present1: Ignored present parameters 000000000fcbfb78.
332:fixme:d3d12_swapchain_Present1: Ignored present parameters 000000000fcbfb78.
332:fixme:d3d12_swapchain_Present1: Ignored present parameters 000000000fcbfb78.

Messages in the logs.

Then, it's as if while the Game menu is running in D3D12, the cutscenes and movies switch back to D3D11:

![image](https://user-images.githubusercontent.com/3923810/121932848-a2a39880-cd45-11eb-8486-86eb56b4e363.png)

When I'm about to begin a new game, the screen becomes black and the crashes soon after as (I suppose) the Opening cutscene is about to start playing:

```
336:fixme:d3d12_swapchain_Present1: Ignored present parameters 000000000fbafc88.
info:  Game: KINGDOM HEARTS FINAL MIX.exe
info:  DXVK: v1.8.1
info:  Built-in extension providers:
info:    Win32 WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
WARNING: radv is not a conformant vulkan implementation, testing use only.
skipping config: /home/darkeox/.config/MangoHud/wine-KINGDOM HEARTS FINAL MIX.conf [ not found ]
skipping config: /home/darkeox/.local/share/lutris/runners/wine/lutris-6.10-x86_64/bin/MangoHud.conf [ not found ]
skipping config: /home/darkeox/.config/MangoHud/wine64-preloader.conf [ not found ]
parsing config: /home/darkeox/.config/MangoHud/MangoHud.conf [ ok ]
info:  AMD RADV SIENNA_CICHLID:
info:    Driver: 21.1.99
info:    Vulkan: 1.2.177
info:    Memory Heap[0]: 
info:      Size: 16128 MiB
info:      Flags: 0x1
info:      Memory Type[0]: Property Flags = 0x1
info:      Memory Type[4]: Property Flags = 0xc1
info:    Memory Heap[1]: 
info:      Size: 16368 MiB
info:      Flags: 0x0
info:      Memory Type[1]: Property Flags = 0x6
info:      Memory Type[3]: Property Flags = 0xe
info:      Memory Type[5]: Property Flags = 0xc6
info:      Memory Type[7]: Property Flags = 0xce
info:    Memory Heap[2]: 
info:      Size: 256 MiB
info:      Flags: 0x1
info:      Memory Type[2]: Property Flags = 0x7
info:      Memory Type[6]: Property Flags = 0xc7
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_1
info:  D3D11CoreCreateDevice: Using feature level D3D_FEATURE_LEVEL_11_1
info:  Device properties:
info:    Device name:     : AMD RADV SIENNA_CICHLID
info:    Driver version   : 21.1.99
info:  Enabled device extensions:
info:    VK_AMD_memory_overallocation_behavior
info:    VK_AMD_shader_fragment_mask
info:    VK_EXT_4444_formats
info:    VK_EXT_custom_border_color
info:    VK_EXT_depth_clip_enable
info:    VK_EXT_extended_dynamic_state
info:    VK_EXT_host_query_reset
info:    VK_EXT_memory_priority
info:    VK_EXT_robustness2
info:    VK_EXT_shader_demote_to_helper_invocation
info:    VK_EXT_shader_stencil_export
info:    VK_EXT_shader_viewport_index_layer
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_create_renderpass2
info:    VK_KHR_depth_stencil_resolve
info:    VK_KHR_draw_indirect_count
info:    VK_KHR_driver_properties
info:    VK_KHR_image_format_list
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_shader_float_controls
info:    VK_KHR_swapchain
info:  Device features:
info:    robustBufferAccess                     : 1
info:    fullDrawIndexUint32                    : 1
info:    imageCubeArray                         : 1
info:    independentBlend                       : 1
info:    geometryShader                         : 1
info:    tessellationShader                     : 1
info:    sampleRateShading                      : 1
info:    dualSrcBlend                           : 1
info:    logicOp                                : 1
info:    multiDrawIndirect                      : 1
info:    drawIndirectFirstInstance              : 1
info:    depthClamp                             : 1
info:    depthBiasClamp                         : 1
info:    fillModeNonSolid                       : 1
info:    depthBounds                            : 1
info:    multiViewport                          : 1
info:    samplerAnisotropy                      : 1
info:    textureCompressionBC                   : 1
info:    occlusionQueryPrecise                  : 1
info:    pipelineStatisticsQuery                : 1
info:    vertexPipelineStoresAndAtomics         : 1
info:    fragmentStoresAndAtomics               : 1
info:    shaderImageGatherExtended              : 1
info:    shaderStorageImageExtendedFormats      : 1
info:    shaderStorageImageReadWithoutFormat    : 1
info:    shaderStorageImageWriteWithoutFormat   : 1
info:    shaderClipDistance                     : 1
info:    shaderCullDistance                     : 1
info:    shaderFloat64                          : 1
info:    shaderInt64                            : 1
info:    variableMultisampleRate                : 1
info:  VK_EXT_4444_formats
info:    formatA4R4G4B4                         : 1
info:    formatA4B4G4R4                         : 1
info:  VK_EXT_custom_border_color
info:    customBorderColors                     : 1
info:    customBorderColorWithoutFormat         : 1
info:  VK_EXT_depth_clip_enable
info:    depthClipEnable                        : 1
info:  VK_EXT_extended_dynamic_state
info:    extendedDynamicState                   : 1
info:  VK_EXT_host_query_reset
info:    hostQueryReset                         : 1
info:  VK_EXT_memory_priority
info:    memoryPriority                         : 1
info:  VK_EXT_robustness2
info:    robustBufferAccess2                    : 1
info:    robustImageAccess2                     : 1
info:    nullDescriptor                         : 1
info:  VK_EXT_shader_demote_to_helper_invocation
info:    shaderDemoteToHelperInvocation         : 1
info:  VK_EXT_transform_feedback
info:    transformFeedback                      : 1
info:    geometryStreams                        : 1
info:  VK_EXT_vertex_attribute_divisor
info:    vertexAttributeInstanceRateDivisor     : 1
info:    vertexAttributeInstanceRateZeroDivisor : 1
info:  Queue families:
info:    Graphics : 0
info:    Transfer : 1
MANGOHUD: Uploading is disabled (permit_upload = 0)
WINE VERSION = wine-6.10.r0.g8a3554ad ( TkG Staging Esync )

info:  DXVK: Read 0 valid state cache entries
info:  DXVK: Using 10 compiler threads
warn:  DXGI: VK_FORMAT_D24_UNORM_S8_UINT -> VK_FORMAT_D32_SFLOAT_S8_UINT
err:   D3D11: Cannot create texture:
err:     Format:  103
err:     Extent:  1920x1080x1
err:     Samples: 1
err:     Layers:  1
err:     Levels:  1
err:     Usage:   0
err:     Flags:   0
err:   D3D11: Cannot create texture:
err:     Format:  103
err:     Extent:  1920x1080x1
err:     Samples: 1
err:     Layers:  1
err:     Levels:  1
err:     Usage:   0
err:     Flags:   0
winegstreamer: warning: videoconvert1: not negotiated
winegstreamer: warning: videoconvert1: ../src-gstreamer/libs/gst/base/gstbasetransform.c(1431): gst_base_transform_reconfigure_unlocked (): /GstBin:bin0/GstVideoConvert:videoconvert1:
not negotiated
wine: Unhandled division by zero at address 00007F1DD5B3022E (thread 0218), starting debugger...
Unhandled exception: divide by zero in 64-bit code (0x00007f1dd5b3022e).
Register dump:
 rip:00007f1dd5b3022e rsp:00000000389afc80 rbp:00000000389afdf8 eflags:00010246 (  R- --  I  Z- -P- )
 rax:0000000000002000 rbx:0000000012843650 rcx:0000000000000000 rdx:0000000000000000
 rsi:0000000012b8ad90 rdi:0000000000000000  r8:0000000000002000  r9:0000000000000000 r10:0000000000000000
 r11:0000000000000000 r12:0000000000000000 r13:0000000000000000 r14:0000000000000000 r15:0000000000000000
Stack dump:
0x00000000389afc80:  0000000000000000 0000000000000000
0x00000000389afc90:  00000000389afde0 00000000389afdf8
0x00000000389afca0:  00000000389afdb0 0000000000000000
0x00000000389afcb0:  0000000012843350 0000000000000000
0x00000000389afcc0:  0000000000000000 00007f1dd5b68cf8
0x00000000389afcd0:  0000000000000000 0000000000000000
0x00000000389afce0:  0000000000000000 0000000000000000
0x00000000389afcf0:  0000000000000000 0000000000000000
0x00000000389afd00:  0000000000000000 0000000000000000
0x00000000389afd10:  0000000000000000 0000000000000000
0x00000000389afd20:  0000000000000000 0000000000000000
0x00000000389afd30:  0000000000000000 0000000000000000
Backtrace:
=>0 0x00007f1dd5b3022e (0x00000000389afdf8)
0x00007f1dd5b3022e: divl	%r11d,%eax
Modules:
Module	Address					Debug info	Name (122 modules)
PE	          630000-          646000	Deferred        vcruntime140
PE	          650000-          654000	Deferred        api-ms-win-crt-runtime-l1-1-0
PE	          660000-          74c000	Deferred        ucrtbase
PE	          750000-          753000	Deferred        api-ms-win-crt-heap-l1-1-0
PE	          760000-          764000	Deferred        api-ms-win-crt-string-l1-1-0
PE	          770000-          774000	Deferred        api-ms-win-crt-stdio-l1-1-0
PE	          780000-          784000	Deferred        api-ms-win-crt-convert-l1-1-0
PE	          790000-          793000	Deferred        api-ms-win-crt-locale-l1-1-0
PE	          7a0000-          7a3000	Deferred        api-ms-win-crt-filesystem-l1-1-0
PE	          7b0000-          7b3000	Deferred        api-ms-win-crt-time-l1-1-0
PE	          7c0000-          7c3000	Deferred        api-ms-win-crt-environment-l1-1-0
PE	          7d0000-          7d5000	Deferred        api-ms-win-crt-math-l1-1-0
PE	          7e0000-          7e3000	Deferred        api-ms-win-crt-utility-l1-1-0
PE	          7f0000-         14e0000	Deferred        eossdk-win64-shipping
PE	         14e0000-         14e3000	Deferred        api-ms-win-crt-conio-l1-1-0
PE	         14f0000-         1572000	Deferred        codex64
PE	         22e0000-         2741000	Deferred        libmono-2.0-x86_64
PE	         4550000-         49c4000	Deferred        mscorlib
PE	         7bd0000-         7e52000	Deferred        system
PE	         7e60000-         7ee0000	Deferred        system.drawing
PE	         7ee0000-         7ee8000	Deferred        accessibility
PE	         7f00000-         7f24000	Deferred        system.configuration
PE	         f170000-         f47e000	Deferred        system.xml
PE	        10000000-        1078e000	Deferred        system.windows.forms
PE	        6a340000-        6a613000	Deferred        d3d11
PE	        6f200000-        6f3e5000	Deferred        dxgi
PE	        6f7c0000-        6fa0f000	Deferred        d3d12
PE	        7b000000-        7b0d7000	Deferred        kernelbase
PE	        7b600000-        7b812000	Deferred        kernel32
PE	        7bc00000-        7bc9e000	Deferred        ntdll
PE	       140000000-       142f8d000	Deferred        kingdom hearts final mix
PE	       180000000-       180091000	Deferred        msvcp140
PE	       1c8890000-       1c88f9000	Deferred        d3dcompiler_47
PE	       1c8b40000-       1c8b60000	Deferred        msacm32
PE	       1c8db0000-       1c8e42000	Deferred        msvcrt
PE	       1cd360000-       1cd364000	Deferred        api-ms-win-core-file-l1-1-0
PE	       1d14b0000-       1d14b3000	Deferred        api-ms-win-core-util-l1-1-0
PE	       1d8e40000-       1d8e43000	Deferred        api-ms-win-core-file-l1-2-0
PE	       1d97a0000-       1d97a7000	Deferred        api-ms-win-core-fibers-l1-1-1
PE	       1dd3f0000-       1dd4bd000	Deferred        crypt32
PE	       1e7610000-       1e7617000	Deferred        normaliz
PE	       1f51e0000-       1f51ef000	Deferred        hid
PE	       1fc170000-       1fc185000	Deferred        fusion
PE	       1fdfd0000-       1fdff7000	Deferred        wintrust
PE	       201cf0000-       201d0f000	Deferred        secur32
PE	       20fc60000-       20fcc8000	Deferred        mf
PE	       212bb0000-       212bb3000	Deferred        api-ms-win-core-processthreads-l1-1-1
PE	       216730000-       216733000	Deferred        api-ms-win-core-handle-l1-1-0
PE	       21a7e0000-       21a855000	Deferred        setupapi
PE	       21b8f0000-       21b8f3000	Deferred        api-ms-win-core-namedpipe-l1-1-0
PE	       226310000-       226313000	Deferred        api-ms-win-core-string-l1-1-0
PE	       228450000-       22847b000	Deferred        propsys
PE	       231ae0000-       231b62000	Deferred        rpcrt4
PE	       236370000-       2363e4000	Deferred        winepulse
PE	       23d820000-       23da4d000	Deferred        user32
PE	       243540000-       2435c5000	Deferred        gdiplus
PE	       24dc40000-       24dc48000	Deferred        api-ms-win-appmodel-runtime-l1-1-1
PE	       24f470000-       24f48f000	Deferred        mpr
PE	       256480000-       256483000	Deferred        api-ms-win-core-datetime-l1-1-0
PE	       263a00000-       263a03000	Deferred        api-ms-win-core-sysinfo-l1-1-0
PE	       26b4c0000-       26b63f000	Deferred        gdi32
PE	       2739c0000-       273af6000	Deferred        oleaut32
PE	       274a50000-       274a5d000	Deferred        kerberos
PE	       280ce0000-       280ce3000	Deferred        api-ms-win-core-interlocked-l1-1-0
PE	       28d6c0000-       28d6c3000	Deferred        api-ms-win-core-memory-l1-1-0
PE	       28ebe0000-       28ebe8000	Deferred        api-ms-win-core-sysinfo-l1-2-1
PE	       297900000-       297912000	Deferred        aclui
PE	       29cfc0000-       29cfd5000	Deferred        dnsapi
PE	       29ea80000-       29ea87000	Deferred        api-ms-win-core-datetime-l1-1-1
PE	       2bb750000-       2bb893000	Deferred        comctl32
PE	       2bc640000-       2bc650000	Deferred        imagehlp
PE	       2c83b0000-       2c83fb000	Deferred        winegstreamer
PE	       2d4d40000-       2d4d57000	Deferred        bcrypt
PE	       2d5140000-       2d5143000	Deferred        api-ms-win-core-file-l2-1-0
PE	       2de970000-       2de9a2000	Deferred        rsaenh
PE	       2e1360000-       2e1363000	Deferred        api-ms-win-core-processenvironment-l1-1-0
PE	       2e3540000-       2e3591000	Deferred        shlwapi
PE	       2e8f10000-       2e9028000	Deferred        ole32
PE	       2f1fa0000-       2f1fad000	Deferred        version
PE	       2f7230000-       2f725d000	Deferred        uxtheme
PE	       2f7f20000-       2f7f2b000	Deferred        vcruntime140_1
PE	       2fa030000-       2fa037000	Deferred        api-ms-win-core-localization-obsolete-l1-2-0
PE	       2fa6b0000-       2fa6b3000	Deferred        api-ms-win-core-console-l1-1-0
PE	       3001a0000-       3001a3000	Deferred        api-ms-win-core-processthreads-l1-1-0
PE	       306e10000-       306e13000	Deferred        api-ms-win-core-timezone-l1-1-0
PE	       30c980000-       30c983000	Deferred        api-ms-win-core-synch-l1-2-0
PE	       3126f0000-       312709000	Deferred        shcore
PE	       319900000-       31992f000	Deferred        mfreadwrite
PE	       327020000-       327072000	Deferred        combase
PE	       32a700000-       32a729000	Deferred        sechost
PE	       330260000-       33029f000	Deferred        advapi32
PE	       335260000-       335272000	Deferred        rtworkq
PE	       336850000-       33687b000	Deferred        mmdevapi
PE	       33a7c0000-       33a7c3000	Deferred        api-ms-win-core-profile-l1-1-0
PE	       33f0c0000-       33f0c3000	Deferred        api-ms-win-core-heap-l1-1-0
PE	       346ba0000-       346bc0000	Deferred        jsproxy
PE	       351680000-       351688000	Deferred        api-ms-win-core-localization-l1-2-1
PE	       353520000-       353523000	Deferred        api-ms-win-core-localization-l1-2-0
PE	       356770000-       3567a9000	Deferred        mscoree
PE	       358120000-       358164000	Deferred        winhttp
PE	       361450000-       361453000	Deferred        api-ms-win-core-synch-l1-1-0
PE	       3673d0000-       3673f7000	Deferred        netapi32
PE	       376c50000-       376c53000	Deferred        api-ms-win-core-errorhandling-l1-1-0
PE	       3785e0000-       3785ec000	Deferred        vulkan-1
PE	       38b5a0000-       38b5f1000	Deferred        wldap32
PE	       390d90000-       390d93000	Deferred        api-ms-win-core-libraryloader-l1-1-0
PE	       39c380000-       39c391000	Deferred        msv1_0
PE	       39cce0000-       39cce3000	Deferred        api-ms-win-core-rtlsupport-l1-1-0
PE	       3a0440000-       3a04bf000	Deferred        wininet
PE	       3a2900000-       3a2903000	Deferred        api-ms-win-core-debug-l1-1-0
PE	       3a6d40000-       3a6d9d000	Deferred        mfplat
PE	       3a93b0000-       3a93bf000	Deferred        xinput1_4
PE	       3afd00000-       3afd1a000	Deferred        imm32
PE	       3b6dc0000-       3b6ddf000	Deferred        winevulkan
PE	       3b8f00000-       3b8fb7000	Deferred        winmm
PE	       3be590000-       3be5fa000	Deferred        dbghelp
PE	    7f1dd49e0000-    7f1dd49e4000	Deferred        winex11
PE	    7f1dd5b60000-    7f1dd5b64000	Deferred        xaudio2_9
PE	    7f1dd5b80000-    7f1dd5b8b000	Deferred        dinput8
PE	    7f1dd5be0000-    7f1dd5be4000	Deferred        iphlpapi
PE	    7f1dd5c10000-    7f1dd5c16000	Deferred        ws2_32
PE	    7f1dd5d60000-    7f1dd6656000	Deferred        shell32
Threads:
process  tid      prio (all id:s are in hex)
00000038 services.exe
	["C:\windows\system32\services.exe"]
	0000003c    0
	00000040    0
	0000004c    0
	00000068    0
	00000080    0
	000000a0    0
	000000b8    0
	000000e4    0
00000044 winedevice.exe
	[C:\windows\system32\winedevice.exe]
	00000048    0
	00000054    0
	00000058    0
	0000005c    0
00000060 plugplay.exe
	[C:\windows\system32\plugplay.exe]
	00000064    0
	0000006c    0
	00000070    0
	00000074    0
	0000008c    0
	00000090    0
	0000013c    0
00000078 winedevice.exe
	[C:\windows\system32\winedevice.exe]
	0000007c    0
	00000084    0
	00000088    0
	00000094    0
	00000098    0
	0000009c    0
	000000b0    0
	000000b4    0
000000a8 svchost.exe
	[C:\windows\system32\svchost.exe -k LocalServiceNetworkRestricted]
	000000ac    0
	000000bc    0
	000000c0    0
000000c4 explorer.exe
	[C:\windows\system32\explorer.exe /desktop]
	000000c8    0
	000000d0    0
	000000d4    0
000000dc rpcss.exe
	[C:\windows\system32\rpcss.exe]
	000000e0    0
	000000e8    0
	000000ec    0
	000000f0    0
	000000f4    0
	000000f8    0
00000100 conhost.exe
	["C:\windows\system32\conhost.exe" --unix --width 0 --height 0 --server 0x34]
	00000104    0
00000110 (D) Z:\opt\zfs_ssd_games\standalone\wine_games\Kingdom Hearts HD ReMIX\KINGDOM HEARTS FINAL MIX.exe
	["Z:\opt\zfs_ssd_games\standalone\wine_games\Kingdom Hearts HD ReMIX\KINGDOM HEARTS FINAL MIX.exe"  -AUTH_LOGIN=unused -AUTH_PASSWORD=0c8814fca1c71b3738a02b639f3b2509 -AUTH_TYPE=exchangecode -epicapp=68c214c58f694ae88c2dab6f209b43e4 -epicenv=Prod -EpicPortal -epicusername=CODEX -epicuserid=1638 -epiclocale=en]
	00000114    1
	00000118    0
	0000011c    0
	00000120    0
	00000124    1
	00000128    1
	0000012c    0
	00000130    0
	00000134    0
	00000138   15
	00000140    0
	00000144    0
	00000148    0
	0000014c    0
	00000150    0
	00000154   15
	00000158   15
	00000160    0
	00000164    0
	00000168    0
	0000016c    0
	00000170    0
	00000174   -1
	00000178    0
	000001c8   -2
	000001cc   -2
	000001d0   -2
	000001d4   -2
	000001d8   -2
	000001dc   -2
	000001e0   -2
	000001e4   -2
	000001e8   -2
	000001ec   -2
	000001f0    0
	000001f4    0
	000001f8    0
	000001fc    0
	00000200    0
	00000204    0
	00000208    0
	0000020c    0
	00000210    0
	00000214    0
	00000218   15 <==
System information:
    Wine build: wine-6.10.r0.g8a3554ad ( TkG Staging Esync )
    Platform: x86_64
    Version: Windows 10
    Host system: Linux
    Host version: 5.12.9-zen1-1-zen
0104:fixme:ver:GetCurrentPackageId (000000000021FDB0 0000000000000000): stub
Monitored process exited.
All monitored processes have exited.
Exit with returncode 0

```

**Comment by DistantThunder** [comment]
If you (re)move the video files, making them unavailable to the game:

![image](https://user-images.githubusercontent.com/3923810/121970166-3ab97600-cd76-11eb-8949-24f23ebabb82.png)

You can go ingame, and indeed it's D3D12 again.

**Comment by Blisto91** [comment]
@arcogabbo @DistantThunder Friendly ping. Does this work now? I understand from above this might not have been a vkd3d-proton issue?

**Comment by Blisto91** [comment]
@arcogabbo @DistantThunder Another friendly ping

**Comment by Kurumi78** [comment]
It was fixed in VKD3D about 6 months ago. This can be closed.
